File: app/src/main/java/cdreyfus/xebia_henri_potier/basket/promotion/CommercialOffersResponse.java
package cdreyfus.xebia_henri_potier.basket.promotion;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class CommercialOffersResponse {

    @SerializedName("offers")
    private
    List<CommercialOffer> commercialOffers;

    public CommercialOffersResponse(){
        commercialOffers = new ArrayList<>();
    }

    public  List<CommercialOffer>  getCommercialOffers() {
        return commercialOffers;
    }

    public void addOffer(CommercialOffer commercialOffer){
        commercialOffers.add(commercialOffer);
    }
}

File: app/src/main/java/cdreyfus/xebia_henri_potier/basket/promotion/CommercialOffersResponse.kt
package cdreyfus.xebia_henri_potier.basket.promotion

import com.google.gson.annotations.SerializedName

import java.util.ArrayList

class CommercialOffersResponse {

    @SerializedName("offers")
    private val commercialOffers: MutableList<CommercialOffer>

    init {
        commercialOffers = ArrayList()
    }

    fun getCommercialOffers(): List<CommercialOffer> {
        return commercialOffers
    }

    fun addOffer(commercialOffer: CommercialOffer) {
        commercialOffers.add(commercialOffer)
    }
}
